<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.6.42">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>QC data on Spectronaut BGS report</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for syntax highlighting */
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { display: inline-block; text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
  }
pre.numberSource { margin-left: 3em;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
</style>


<script src="DataAnalysis_files/libs/clipboard/clipboard.min.js"></script>
<script src="DataAnalysis_files/libs/quarto-html/quarto.js"></script>
<script src="DataAnalysis_files/libs/quarto-html/popper.min.js"></script>
<script src="DataAnalysis_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="DataAnalysis_files/libs/quarto-html/anchor.min.js"></script>
<link href="DataAnalysis_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="DataAnalysis_files/libs/quarto-html/quarto-syntax-highlighting-2f5df379a58b258e96c21c0638c20c03.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="DataAnalysis_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="DataAnalysis_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="DataAnalysis_files/libs/bootstrap/bootstrap-1bc8a17f135ab3d594c857e9f48e611b.min.css" rel="stylesheet" append-hash="true" id="quarto-bootstrap" data-mode="light">


</head>

<body>

<div id="quarto-content" class="page-columns page-rows-contents page-layout-article">
<div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
<div class="quarto-alternate-formats"><h2>Other Formats</h2><ul><li><a href="DataAnalysis.pdf"><i class="bi bi-file-pdf"></i>PDF</a></li></ul></div></div>
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">QC data on Spectronaut BGS report</h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  


</header>


<section id="load-data-and-packages" class="level2">
<h2 class="anchored" data-anchor-id="load-data-and-packages">Load data and packages</h2>
<p>Data is loaded from Spectronaut BGS Extended report. Report contains peptide and protein information.</p>
<div class="cell">
<details class="code-fold">
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb1"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a><span class="fu">source</span>(<span class="st">"source.R"</span>)</span>
<span id="cb1-2"><a href="#cb1-2" aria-hidden="true" tabindex="-1"></a><span class="fu">source</span>(<span class="st">"functions.R"</span>)</span>
<span id="cb1-3"><a href="#cb1-3" aria-hidden="true" tabindex="-1"></a>file <span class="ot">=</span> <span class="st">"raw/20250703_073723_StemCells_ANWR_Run5_Plasma_PE_Report.tsv"</span></span>
<span id="cb1-4"><a href="#cb1-4" aria-hidden="true" tabindex="-1"></a>dfBGS <span class="ot">&lt;-</span> data.table<span class="sc">::</span><span class="fu">fread</span>(file)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
<section id="calculate-search-summaries" class="level3">
<h3 class="anchored" data-anchor-id="calculate-search-summaries">Calculate search summaries</h3>
<div class="cell">
<details class="code-fold">
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb2"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb2-1"><a href="#cb2-1" aria-hidden="true" tabindex="-1"></a><span class="fu">data.frame</span>(<span class="st">"File"</span> <span class="ot">=</span> file,</span>
<span id="cb2-2"><a href="#cb2-2" aria-hidden="true" tabindex="-1"></a>           <span class="st">"Size, GB"</span> <span class="ot">=</span> <span class="fu">file.info</span>(file)<span class="sc">$</span>size<span class="sc">/</span><span class="dv">1024</span><span class="sc">^</span><span class="dv">3</span>,</span>
<span id="cb2-3"><a href="#cb2-3" aria-hidden="true" tabindex="-1"></a>           <span class="st">"Date"</span> <span class="ot">=</span> <span class="fu">file.info</span>(file)<span class="sc">$</span>mtime,</span>
<span id="cb2-4"><a href="#cb2-4" aria-hidden="true" tabindex="-1"></a>           <span class="st">"FASTA"</span> <span class="ot">=</span> dfBGS <span class="sc">%&gt;%</span> <span class="fu">distinct</span>(E.LFQMethod, PG.FASTAName) <span class="sc">%&gt;%</span> <span class="fu">group_by</span>(PG.FASTAName) <span class="sc">%&gt;%</span>  <span class="fu">summarise</span>(<span class="at">n =</span> <span class="fu">paste</span>(PG.FASTAName, <span class="at">collapse =</span> <span class="st">";"</span>)) <span class="sc">%&gt;%</span> <span class="fu">pull</span>(n),</span>
<span id="cb2-5"><a href="#cb2-5" aria-hidden="true" tabindex="-1"></a>           <span class="st">"Species"</span> <span class="ot">=</span> dfBGS <span class="sc">%&gt;%</span> <span class="fu">distinct</span>(E.LFQMethod, PEP.AllOccurringOrganisms) <span class="sc">%&gt;%</span> <span class="fu">group_by</span>(E.LFQMethod) <span class="sc">%&gt;%</span>  <span class="fu">summarise</span>(<span class="at">n =</span> <span class="fu">paste</span>(PEP.AllOccurringOrganisms, <span class="at">collapse =</span> <span class="st">";"</span>)) <span class="sc">%&gt;%</span> <span class="fu">pull</span>(n),</span>
<span id="cb2-6"><a href="#cb2-6" aria-hidden="true" tabindex="-1"></a>           <span class="st">"Errors"</span> <span class="ot">=</span> dfBGS <span class="sc">%&gt;%</span> <span class="fu">head</span>(<span class="dv">1</span>) <span class="sc">%&gt;%</span> <span class="fu">pull</span>(E.Errors),</span>
<span id="cb2-7"><a href="#cb2-7" aria-hidden="true" tabindex="-1"></a>           <span class="st">"Warnings"</span> <span class="ot">=</span> dfBGS <span class="sc">%&gt;%</span> <span class="fu">head</span>(<span class="dv">1</span>) <span class="sc">%&gt;%</span> <span class="fu">pull</span>(E.Warnings),</span>
<span id="cb2-8"><a href="#cb2-8" aria-hidden="true" tabindex="-1"></a>           <span class="st">"Files"</span> <span class="ot">=</span> dfBGS <span class="sc">%&gt;%</span> <span class="fu">distinct</span>(R.FileName) <span class="sc">%&gt;%</span> <span class="fu">nrow</span>(),</span>
<span id="cb2-9"><a href="#cb2-9" aria-hidden="true" tabindex="-1"></a>           <span class="st">"Conditions"</span> <span class="ot">=</span> dfBGS <span class="sc">%&gt;%</span> <span class="fu">distinct</span>(R.Condition) <span class="sc">%&gt;%</span> <span class="fu">nrow</span>()) <span class="ot">-&gt;</span> r</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
<p><strong>Search summary:</strong> Search file raw/20250703_073723_StemCells_ANWR_Run5_Plasma_PE_Report.tsv, raw/20250703_073723_StemCells_ANWR_Run5_Plasma_PE_Report.tsv, raw/20250703_073723_StemCells_ANWR_Run5_Plasma_PE_Report.tsv, raw/20250703_073723_StemCells_ANWR_Run5_Plasma_PE_Report.tsv, raw/20250703_073723_StemCells_ANWR_Run5_Plasma_PE_Report.tsv, raw/20250703_073723_StemCells_ANWR_Run5_Plasma_PE_Report.tsv, raw/20250703_073723_StemCells_ANWR_Run5_Plasma_PE_Report.tsv, raw/20250703_073723_StemCells_ANWR_Run5_Plasma_PE_Report.tsv, raw/20250703_073723_StemCells_ANWR_Run5_Plasma_PE_Report.tsv, raw/20250703_073723_StemCells_ANWR_Run5_Plasma_PE_Report.tsv, raw/20250703_073723_StemCells_ANWR_Run5_Plasma_PE_Report.tsv, raw/20250703_073723_StemCells_ANWR_Run5_Plasma_PE_Report.tsv, raw/20250703_073723_StemCells_ANWR_Run5_Plasma_PE_Report.tsv, raw/20250703_073723_StemCells_ANWR_Run5_Plasma_PE_Report.tsv, raw/20250703_073723_StemCells_ANWR_Run5_Plasma_PE_Report.tsv, raw/20250703_073723_StemCells_ANWR_Run5_Plasma_PE_Report.tsv, raw/20250703_073723_StemCells_ANWR_Run5_Plasma_PE_Report.tsv, raw/20250703_073723_StemCells_ANWR_Run5_Plasma_PE_Report.tsv, raw/20250703_073723_StemCells_ANWR_Run5_Plasma_PE_Report.tsv, raw/20250703_073723_StemCells_ANWR_Run5_Plasma_PE_Report.tsv, raw/20250703_073723_StemCells_ANWR_Run5_Plasma_PE_Report.tsv, raw/20250703_073723_StemCells_ANWR_Run5_Plasma_PE_Report.tsv, raw/20250703_073723_StemCells_ANWR_Run5_Plasma_PE_Report.tsv, raw/20250703_073723_StemCells_ANWR_Run5_Plasma_PE_Report.tsv, raw/20250703_073723_StemCells_ANWR_Run5_Plasma_PE_Report.tsv, raw/20250703_073723_StemCells_ANWR_Run5_Plasma_PE_Report.tsv, raw/20250703_073723_StemCells_ANWR_Run5_Plasma_PE_Report.tsv, raw/20250703_073723_StemCells_ANWR_Run5_Plasma_PE_Report.tsv, raw/20250703_073723_StemCells_ANWR_Run5_Plasma_PE_Report.tsv, raw/20250703_073723_StemCells_ANWR_Run5_Plasma_PE_Report.tsv, raw/20250703_073723_StemCells_ANWR_Run5_Plasma_PE_Report.tsv, raw/20250703_073723_StemCells_ANWR_Run5_Plasma_PE_Report.tsv, raw/20250703_073723_StemCells_ANWR_Run5_Plasma_PE_Report.tsv, raw/20250703_073723_StemCells_ANWR_Run5_Plasma_PE_Report.tsv, raw/20250703_073723_StemCells_ANWR_Run5_Plasma_PE_Report.tsv, raw/20250703_073723_StemCells_ANWR_Run5_Plasma_PE_Report.tsv, raw/20250703_073723_StemCells_ANWR_Run5_Plasma_PE_Report.tsv, raw/20250703_073723_StemCells_ANWR_Run5_Plasma_PE_Report.tsv, raw/20250703_073723_StemCells_ANWR_Run5_Plasma_PE_Report.tsv, raw/20250703_073723_StemCells_ANWR_Run5_Plasma_PE_Report.tsv, raw/20250703_073723_StemCells_ANWR_Run5_Plasma_PE_Report.tsv, raw/20250703_073723_StemCells_ANWR_Run5_Plasma_PE_Report.tsv (1.0155299, 1.0155299, 1.0155299, 1.0155299, 1.0155299, 1.0155299, 1.0155299, 1.0155299, 1.0155299, 1.0155299, 1.0155299, 1.0155299, 1.0155299, 1.0155299, 1.0155299, 1.0155299, 1.0155299, 1.0155299, 1.0155299, 1.0155299, 1.0155299, 1.0155299, 1.0155299, 1.0155299, 1.0155299, 1.0155299, 1.0155299, 1.0155299, 1.0155299, 1.0155299, 1.0155299, 1.0155299, 1.0155299, 1.0155299, 1.0155299, 1.0155299, 1.0155299, 1.0155299, 1.0155299, 1.0155299, 1.0155299, 1.0155299GB) is last modified 2025-07-03 10:11:19.168236, 2025-07-03 10:11:19.168236, 2025-07-03 10:11:19.168236, 2025-07-03 10:11:19.168236, 2025-07-03 10:11:19.168236, 2025-07-03 10:11:19.168236, 2025-07-03 10:11:19.168236, 2025-07-03 10:11:19.168236, 2025-07-03 10:11:19.168236, 2025-07-03 10:11:19.168236, 2025-07-03 10:11:19.168236, 2025-07-03 10:11:19.168236, 2025-07-03 10:11:19.168236, 2025-07-03 10:11:19.168236, 2025-07-03 10:11:19.168236, 2025-07-03 10:11:19.168236, 2025-07-03 10:11:19.168236, 2025-07-03 10:11:19.168236, 2025-07-03 10:11:19.168236, 2025-07-03 10:11:19.168236, 2025-07-03 10:11:19.168236, 2025-07-03 10:11:19.168236, 2025-07-03 10:11:19.168236, 2025-07-03 10:11:19.168236, 2025-07-03 10:11:19.168236, 2025-07-03 10:11:19.168236, 2025-07-03 10:11:19.168236, 2025-07-03 10:11:19.168236, 2025-07-03 10:11:19.168236, 2025-07-03 10:11:19.168236, 2025-07-03 10:11:19.168236, 2025-07-03 10:11:19.168236, 2025-07-03 10:11:19.168236, 2025-07-03 10:11:19.168236, 2025-07-03 10:11:19.168236, 2025-07-03 10:11:19.168236, 2025-07-03 10:11:19.168236, 2025-07-03 10:11:19.168236, 2025-07-03 10:11:19.168236, 2025-07-03 10:11:19.168236, 2025-07-03 10:11:19.168236, 2025-07-03 10:11:19.168236.<br>
Search contains 61, 61, 61, 61, 61, 61, 61, 61, 61, 61, 61, 61, 61, 61, 61, 61, 61, 61, 61, 61, 61, 61, 61, 61, 61, 61, 61, 61, 61, 61, 61, 61, 61, 61, 61, 61, 61, 61, 61, 61, 61, 61 files across 22, 22, 22, 22, 22, 22, 22, 22, 22, 22, 22, 22, 22, 22, 22, 22, 22, 22, 22, 22, 22, 22, 22, 22, 22, 22, 22, 22, 22, 22, 22, 22, 22, 22, 22, 22, 22, 22, 22, 22, 22, 22 conditions.<br>
Files are searched against database Human_uniprotkb_9606_ARWN_Modified_20250527, Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Pig_Proteome_UP000008227_9823, Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Pig_Proteome_UP000008227_9823, Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Pig_Proteome_UP000008227_9823, Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Pig_Proteome_UP000008227_9823, Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Pig_Proteome_UP000008227_9823;Human_uniprotkb_9606_ARWN_Modified_20250527, Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Pig_Proteome_UP000008227_9823;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527, Human_uniprotkb_9606_ARWN_Modified_20250527;Pig_Proteome_UP000008227_9823, Human_uniprotkb_9606_ARWN_Modified_20250527;Pig_Proteome_UP000008227_9823;Human_uniprotkb_9606_ARWN_Modified_20250527, Human_uniprotkb_9606_ARWN_Modified_20250527;Pig_Proteome_UP000008227_9823;Pig_Proteome_UP000008227_9823;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527, Pig_Proteome_UP000008227_9823, Pig_Proteome_UP000008227_9823;Human_uniprotkb_9606_ARWN_Modified_20250527, Pig_Proteome_UP000008227_9823;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527, Pig_Proteome_UP000008227_9823;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527, Pig_Proteome_UP000008227_9823;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527, Pig_Proteome_UP000008227_9823;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527, Pig_Proteome_UP000008227_9823;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527, Pig_Proteome_UP000008227_9823;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527, Pig_Proteome_UP000008227_9823;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527, Pig_Proteome_UP000008227_9823;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527, Pig_Proteome_UP000008227_9823;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527, Pig_Proteome_UP000008227_9823;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527, Pig_Proteome_UP000008227_9823;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Pig_Proteome_UP000008227_9823, Pig_Proteome_UP000008227_9823;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Pig_Proteome_UP000008227_9823, Pig_Proteome_UP000008227_9823;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Pig_Proteome_UP000008227_9823, Pig_Proteome_UP000008227_9823;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Pig_Proteome_UP000008227_9823, Pig_Proteome_UP000008227_9823;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Pig_Proteome_UP000008227_9823;Human_uniprotkb_9606_ARWN_Modified_20250527, Pig_Proteome_UP000008227_9823;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Pig_Proteome_UP000008227_9823, Pig_Proteome_UP000008227_9823;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Pig_Proteome_UP000008227_9823, Pig_Proteome_UP000008227_9823;Human_uniprotkb_9606_ARWN_Modified_20250527;Pig_Proteome_UP000008227_9823, Pig_Proteome_UP000008227_9823;Pig_Proteome_UP000008227_9823;Human_uniprotkb_9606_ARWN_Modified_20250527, Pig_Proteome_UP000008227_9823;Pig_Proteome_UP000008227_9823;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527, Pig_Proteome_UP000008227_9823;Pig_Proteome_UP000008227_9823;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527, Pig_Proteome_UP000008227_9823;Pig_Proteome_UP000008227_9823;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527, Pig_Proteome_UP000008227_9823;Pig_Proteome_UP000008227_9823;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527, Pig_Proteome_UP000008227_9823;Pig_Proteome_UP000008227_9823;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Pig_Proteome_UP000008227_9823, Pig_Proteome_UP000008227_9823;Pig_Proteome_UP000008227_9823;Pig_Proteome_UP000008227_9823;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527, Pig_Proteome_UP000008227_9823;Pig_Proteome_UP000008227_9823;Pig_Proteome_UP000008227_9823;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527, Pig_Proteome_UP000008227_9823;Pig_Proteome_UP000008227_9823;Pig_Proteome_UP000008227_9823;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Pig_Proteome_UP000008227_9823, Pig_Proteome_UP000008227_9823;Pig_Proteome_UP000008227_9823;Pig_Proteome_UP000008227_9823;Human_uniprotkb_9606_ARWN_Modified_20250527;Pig_Proteome_UP000008227_9823, Pig_Proteome_UP000008227_9823;Pig_Proteome_UP000008227_9823;Pig_Proteome_UP000008227_9823;Pig_Proteome_UP000008227_9823;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Pig_Proteome_UP000008227_9823;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Pig_Proteome_UP000008227_9823, Pig_Proteome_UP000008227_9823;Pig_Proteome_UP000008227_9823;Pig_Proteome_UP000008227_9823;Pig_Proteome_UP000008227_9823;Pig_Proteome_UP000008227_9823;Pig_Proteome_UP000008227_9823;Pig_Proteome_UP000008227_9823;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527;Human_uniprotkb_9606_ARWN_Modified_20250527(Homo sapiens;Sus scrofa;Sus scrofa;Homo sapiens, Homo sapiens;Sus scrofa;Sus scrofa;Homo sapiens, Homo sapiens;Sus scrofa;Sus scrofa;Homo sapiens, Homo sapiens;Sus scrofa;Sus scrofa;Homo sapiens, Homo sapiens;Sus scrofa;Sus scrofa;Homo sapiens, Homo sapiens;Sus scrofa;Sus scrofa;Homo sapiens, Homo sapiens;Sus scrofa;Sus scrofa;Homo sapiens, Homo sapiens;Sus scrofa;Sus scrofa;Homo sapiens, Homo sapiens;Sus scrofa;Sus scrofa;Homo sapiens, Homo sapiens;Sus scrofa;Sus scrofa;Homo sapiens, Homo sapiens;Sus scrofa;Sus scrofa;Homo sapiens, Homo sapiens;Sus scrofa;Sus scrofa;Homo sapiens, Homo sapiens;Sus scrofa;Sus scrofa;Homo sapiens, Homo sapiens;Sus scrofa;Sus scrofa;Homo sapiens, Homo sapiens;Sus scrofa;Sus scrofa;Homo sapiens, Homo sapiens;Sus scrofa;Sus scrofa;Homo sapiens, Homo sapiens;Sus scrofa;Sus scrofa;Homo sapiens, Homo sapiens;Sus scrofa;Sus scrofa;Homo sapiens, Homo sapiens;Sus scrofa;Sus scrofa;Homo sapiens, Homo sapiens;Sus scrofa;Sus scrofa;Homo sapiens, Homo sapiens;Sus scrofa;Sus scrofa;Homo sapiens, Homo sapiens;Sus scrofa;Sus scrofa;Homo sapiens, Homo sapiens;Sus scrofa;Sus scrofa;Homo sapiens, Homo sapiens;Sus scrofa;Sus scrofa;Homo sapiens, Homo sapiens;Sus scrofa;Sus scrofa;Homo sapiens, Homo sapiens;Sus scrofa;Sus scrofa;Homo sapiens, Homo sapiens;Sus scrofa;Sus scrofa;Homo sapiens, Homo sapiens;Sus scrofa;Sus scrofa;Homo sapiens, Homo sapiens;Sus scrofa;Sus scrofa;Homo sapiens, Homo sapiens;Sus scrofa;Sus scrofa;Homo sapiens, Homo sapiens;Sus scrofa;Sus scrofa;Homo sapiens, Homo sapiens;Sus scrofa;Sus scrofa;Homo sapiens, Homo sapiens;Sus scrofa;Sus scrofa;Homo sapiens, Homo sapiens;Sus scrofa;Sus scrofa;Homo sapiens, Homo sapiens;Sus scrofa;Sus scrofa;Homo sapiens, Homo sapiens;Sus scrofa;Sus scrofa;Homo sapiens, Homo sapiens;Sus scrofa;Sus scrofa;Homo sapiens, Homo sapiens;Sus scrofa;Sus scrofa;Homo sapiens, Homo sapiens;Sus scrofa;Sus scrofa;Homo sapiens, Homo sapiens;Sus scrofa;Sus scrofa;Homo sapiens, Homo sapiens;Sus scrofa;Sus scrofa;Homo sapiens, Homo sapiens;Sus scrofa;Sus scrofa;Homo sapiens) with 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 errors and 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 warnings.</p>
</section>
<section id="sample-table" class="level3">
<h3 class="anchored" data-anchor-id="sample-table">Sample table</h3>
<p>Summary table of samples. Unique proteins, peptides, mean missed cleavage and summed intensity is calculated.</p>
<div class="cell">
<details class="code-fold">
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb3"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb3-1"><a href="#cb3-1" aria-hidden="true" tabindex="-1"></a>dfBGS <span class="sc">%&gt;%</span> </span>
<span id="cb3-2"><a href="#cb3-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">SummaryTableSamples</span>(<span class="at">exportTable =</span> T) <span class="ot">-&gt;</span> t</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
</section>
<section id="condition-table" class="level3">
<h3 class="anchored" data-anchor-id="condition-table">Condition table</h3>
<p>Summary table calculating files per condition, mean number of proteins, min/max number of proteins, mean number of peptides and min/max number of peptides.</p>
<div class="cell">
<details class="code-fold">
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb4"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb4-1"><a href="#cb4-1" aria-hidden="true" tabindex="-1"></a>dfBGS <span class="sc">%&gt;%</span> </span>
<span id="cb4-2"><a href="#cb4-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">SummaryTableConditions</span>() <span class="ot">-&gt;</span> s</span>
<span id="cb4-3"><a href="#cb4-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb4-4"><a href="#cb4-4" aria-hidden="true" tabindex="-1"></a>s</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output cell-output-stdout">
<pre><code># A tibble: 22 × 6
   Condition         Files MeanProteins RangeProteins MeanPeptides RangePeptides
   &lt;chr&gt;             &lt;int&gt;        &lt;dbl&gt; &lt;chr&gt;                &lt;dbl&gt; &lt;chr&gt;        
 1 pig126_day minus…     3        1060  1009-1157            8726. 8340-9447    
 2 pig126_day1           3        1023  817-1199             8050  6371-9548    
 3 pig126_day14          3        1446  1394-1514           11153. 10673-11864  
 4 pig126_day2           2        1138. 1114-1163            9938. 9896-9979    
 5 pig126_day21          3        1328. 933-2062            10943  8028-16256   
 6 pig126_day28          3        1001. 895-1073             8655. 7213-9417    
 7 pig126_day42          3         849. 458-1102             6957  3600-8724    
 8 pig126_day5           3        1120  1087-1166            8982. 8533-9339    
 9 pig126_day56          3        1099. 1031-1136            9045  8704-9508    
10 pig126_day7           3         987. 809-1274             7134. 6086-9098    
# ℹ 12 more rows</code></pre>
</div>
</div>
</section>
<section id="samples-per-condition" class="level3">
<h3 class="anchored" data-anchor-id="samples-per-condition">Samples per condition</h3>
<p>Bar plot of number of raw files in each condition.</p>
<div class="cell">
<details class="code-fold">
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb6"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb6-1"><a href="#cb6-1" aria-hidden="true" tabindex="-1"></a>dfBGS <span class="sc">%&gt;%</span> </span>
<span id="cb6-2"><a href="#cb6-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">MDBL_SamplesPerCondition</span>()</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="DataAnalysis_files/figure-html/unnamed-chunk-4-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</section>
<section id="missed-cleavages" class="level3">
<h3 class="anchored" data-anchor-id="missed-cleavages">Missed cleavages</h3>
<p>Plot of number of missed cleavages in the peptides identified per sample. Plot is grouped on condition and scaled to 100%.</p>
<div class="cell">
<details class="code-fold">
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb7"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb7-1"><a href="#cb7-1" aria-hidden="true" tabindex="-1"></a>dfBGS <span class="sc">%&gt;%</span> </span>
<span id="cb7-2"><a href="#cb7-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">MDBL_PlotMissedCleavage</span>()</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="DataAnalysis_files/figure-html/unnamed-chunk-5-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</section>
<section id="proteins-per-raw-file" class="level3">
<h3 class="anchored" data-anchor-id="proteins-per-raw-file">Proteins per raw file</h3>
<p>Box plot of number of unique proteins identified per raw file. Plot is grouped on condition.</p>
<div class="cell">
<details class="code-fold">
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb8"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb8-1"><a href="#cb8-1" aria-hidden="true" tabindex="-1"></a>dfBGS <span class="sc">%&gt;%</span> </span>
<span id="cb8-2"><a href="#cb8-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">MDBL_ProteinsPerFileBoxplot</span>()</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="DataAnalysis_files/figure-html/unnamed-chunk-6-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</section>
<section id="proteins-per-condition" class="level3">
<h3 class="anchored" data-anchor-id="proteins-per-condition">Proteins per condition</h3>
<p>Bar plot of total number of unique proteins identified per condition.</p>
<div class="cell">
<details class="code-fold">
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb9"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb9-1"><a href="#cb9-1" aria-hidden="true" tabindex="-1"></a>dfBGS <span class="sc">%&gt;%</span> </span>
<span id="cb9-2"><a href="#cb9-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">distinct</span>(R.Condition, PG.ProteinAccessions) <span class="sc">%&gt;%</span> </span>
<span id="cb9-3"><a href="#cb9-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(R.Condition) <span class="sc">%&gt;%</span> </span>
<span id="cb9-4"><a href="#cb9-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">tally</span>() <span class="sc">%&gt;%</span> </span>
<span id="cb9-5"><a href="#cb9-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ggplot</span>(<span class="fu">aes</span>(n, R.Condition, <span class="at">fill=</span>R.Condition, <span class="at">label=</span>n))<span class="sc">+</span></span>
<span id="cb9-6"><a href="#cb9-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_col</span>()<span class="sc">+</span></span>
<span id="cb9-7"><a href="#cb9-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_text</span>(<span class="at">hjust=</span><span class="fl">1.3</span>)<span class="sc">+</span></span>
<span id="cb9-8"><a href="#cb9-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">title=</span><span class="st">"Unique proteins per condition"</span>, <span class="at">x=</span><span class="st">"unique proteins"</span>, <span class="at">y=</span><span class="st">""</span>)<span class="sc">+</span></span>
<span id="cb9-9"><a href="#cb9-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_minimal</span>()<span class="sc">+</span></span>
<span id="cb9-10"><a href="#cb9-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(<span class="at">legend.position =</span> <span class="st">"none"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="DataAnalysis_files/figure-html/unnamed-chunk-7-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</section>
<section id="proteins-per-raw-file-1" class="level3">
<h3 class="anchored" data-anchor-id="proteins-per-raw-file-1">Proteins per raw file</h3>
<p>Bar plot of unique proteins identified per sample. Plot is grouped on conditions.</p>
<div class="cell">
<details class="code-fold">
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb10"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb10-1"><a href="#cb10-1" aria-hidden="true" tabindex="-1"></a>dfBGS <span class="sc">%&gt;%</span> </span>
<span id="cb10-2"><a href="#cb10-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">MDBL_ProteinsPerFileBarplot</span>()</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="DataAnalysis_files/figure-html/unnamed-chunk-8-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</section>
<section id="summed-intensities" class="level3">
<h3 class="anchored" data-anchor-id="summed-intensities">Summed Intensities</h3>
<div class="cell">
<details class="code-fold">
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb11"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb11-1"><a href="#cb11-1" aria-hidden="true" tabindex="-1"></a>dfBGS <span class="sc">%&gt;%</span> </span>
<span id="cb11-2"><a href="#cb11-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">distinct</span>(R.FileName, R.Condition, PG.ProteinAccessions, PG.Quantity) <span class="sc">%&gt;%</span> </span>
<span id="cb11-3"><a href="#cb11-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(R.FileName, R.Condition) <span class="sc">%&gt;%</span> </span>
<span id="cb11-4"><a href="#cb11-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">summarize</span>(<span class="at">SummedIntensity =</span> <span class="fu">log10</span>(<span class="fu">sum</span>(PG.Quantity, <span class="at">na.rm =</span> T))) <span class="sc">%&gt;%</span> </span>
<span id="cb11-5"><a href="#cb11-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ggplot</span>(<span class="fu">aes</span>(SummedIntensity, R.Condition, <span class="at">fill=</span>R.Condition))<span class="sc">+</span></span>
<span id="cb11-6"><a href="#cb11-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_boxplot</span>(<span class="at">outlier.shape =</span> <span class="cn">NA</span>)<span class="sc">+</span></span>
<span id="cb11-7"><a href="#cb11-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_jitter</span>(<span class="at">alpha=</span><span class="fl">0.3</span>, <span class="at">height =</span> <span class="fl">0.2</span>, <span class="at">pch=</span><span class="dv">16</span>)<span class="sc">+</span></span>
<span id="cb11-8"><a href="#cb11-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">title=</span><span class="st">"Summed intensities per rawfile"</span>, <span class="at">x=</span><span class="st">"Summed intensity [log10]"</span>, <span class="at">y=</span><span class="st">""</span>)<span class="sc">+</span></span>
<span id="cb11-9"><a href="#cb11-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_minimal</span>()<span class="sc">+</span></span>
<span id="cb11-10"><a href="#cb11-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(<span class="at">legend.position =</span> <span class="st">"none"</span>,</span>
<span id="cb11-11"><a href="#cb11-11" aria-hidden="true" tabindex="-1"></a>        <span class="at">panel.grid.minor.y =</span> <span class="fu">element_blank</span>(),</span>
<span id="cb11-12"><a href="#cb11-12" aria-hidden="true" tabindex="-1"></a>        <span class="at">panel.grid.minor.x =</span> <span class="fu">element_blank</span>())</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="DataAnalysis_files/figure-html/unnamed-chunk-9-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</section>
<section id="test-peptides-per-minute" class="level3">
<h3 class="anchored" data-anchor-id="test-peptides-per-minute">test, peptides per minute</h3>
<div class="cell">
<details class="code-fold">
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb12"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb12-1"><a href="#cb12-1" aria-hidden="true" tabindex="-1"></a>dfBGS <span class="sc">%&gt;%</span>  </span>
<span id="cb12-2"><a href="#cb12-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ggplot</span>(<span class="fu">aes</span>(EG.ApexRT, <span class="at">group=</span>R.FileName, <span class="at">color=</span>R.Condition))<span class="sc">+</span></span>
<span id="cb12-3"><a href="#cb12-3" aria-hidden="true" tabindex="-1"></a>  <span class="co">#geom_histogram(binwidth = 0.5)+</span></span>
<span id="cb12-4"><a href="#cb12-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_density</span>() <span class="sc">+</span></span>
<span id="cb12-5"><a href="#cb12-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">facet_wrap</span>(<span class="sc">~</span>R.Condition)<span class="sc">+</span></span>
<span id="cb12-6"><a href="#cb12-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">title=</span><span class="st">"Peptides identified per minutte"</span>, <span class="at">subtitle=</span><span class="st">""</span>, <span class="at">x=</span><span class="st">""</span>, <span class="at">y=</span><span class="st">""</span>)<span class="sc">+</span></span>
<span id="cb12-7"><a href="#cb12-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_minimal</span>()<span class="sc">+</span></span>
<span id="cb12-8"><a href="#cb12-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(<span class="at">legend.position =</span> <span class="st">"none"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="DataAnalysis_files/figure-html/unnamed-chunk-10-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</section>
<section id="peptide-indentifications-per-second" class="level3">
<h3 class="anchored" data-anchor-id="peptide-indentifications-per-second">Peptide indentifications per second</h3>
<div class="cell">
<details class="code-fold">
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb13"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb13-1"><a href="#cb13-1" aria-hidden="true" tabindex="-1"></a>my_bins <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="fu">seq</span>(<span class="dv">0</span>, <span class="dv">12</span>, <span class="at">by=</span><span class="fl">0.0166667</span>))</span>
<span id="cb13-2"><a href="#cb13-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb13-3"><a href="#cb13-3" aria-hidden="true" tabindex="-1"></a><span class="cf">for</span>(n <span class="cf">in</span> dfBGS <span class="sc">%&gt;%</span> <span class="fu">distinct</span>(R.Condition) <span class="sc">%&gt;%</span> <span class="fu">pull</span>(R.Condition)){</span>
<span id="cb13-4"><a href="#cb13-4" aria-hidden="true" tabindex="-1"></a>  dfBGS <span class="sc">%&gt;%</span> </span>
<span id="cb13-5"><a href="#cb13-5" aria-hidden="true" tabindex="-1"></a>    <span class="co">#filter(R.Condition == n) %&gt;% </span></span>
<span id="cb13-6"><a href="#cb13-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">select</span>(R.FileName, R.Condition, EG.ApexRT) <span class="sc">%&gt;%</span> </span>
<span id="cb13-7"><a href="#cb13-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">binned =</span> <span class="fu">cut</span>(EG.ApexRT, my_bins)) <span class="sc">%&gt;%</span> </span>
<span id="cb13-8"><a href="#cb13-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">bin =</span> <span class="fu">str_remove</span>(binned, <span class="st">",.*"</span>)) <span class="sc">%&gt;%</span> </span>
<span id="cb13-9"><a href="#cb13-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">bin =</span> <span class="fu">as.numeric</span>(<span class="fu">str_remove</span>(bin, <span class="st">"[(]"</span> ))) <span class="sc">%&gt;%</span> </span>
<span id="cb13-10"><a href="#cb13-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(R.Condition, R.FileName, bin) <span class="sc">%&gt;%</span></span>
<span id="cb13-11"><a href="#cb13-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">count</span>() <span class="sc">%&gt;%</span>  </span>
<span id="cb13-12"><a href="#cb13-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ggplot</span>(<span class="fu">aes</span>(bin, n, <span class="at">group=</span>R.FileName, <span class="at">color=</span>R.Condition))<span class="sc">+</span></span>
<span id="cb13-13"><a href="#cb13-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_line</span>()<span class="sc">+</span></span>
<span id="cb13-14"><a href="#cb13-14" aria-hidden="true" tabindex="-1"></a>  <span class="fu">facet_wrap</span>(<span class="sc">~</span>R.FileName, <span class="at">ncol=</span><span class="dv">3</span>)<span class="sc">+</span></span>
<span id="cb13-15"><a href="#cb13-15" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">title=</span><span class="fu">paste0</span>(<span class="st">"Peptides identified in "</span>,n, <span class="st">" per second"</span>), <span class="at">subtitle=</span><span class="st">""</span>, <span class="at">x=</span><span class="st">""</span>, <span class="at">y=</span><span class="st">""</span>)<span class="sc">+</span></span>
<span id="cb13-16"><a href="#cb13-16" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_minimal</span>()<span class="sc">+</span></span>
<span id="cb13-17"><a href="#cb13-17" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(<span class="at">legend.position =</span> <span class="st">"none"</span>) <span class="ot">-&gt;</span> p</span>
<span id="cb13-18"><a href="#cb13-18" aria-hidden="true" tabindex="-1"></a>  <span class="fu">plot</span>(p)</span>
<span id="cb13-19"><a href="#cb13-19" aria-hidden="true" tabindex="-1"></a>}</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="DataAnalysis_files/figure-html/unnamed-chunk-11-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="DataAnalysis_files/figure-html/unnamed-chunk-11-2.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="DataAnalysis_files/figure-html/unnamed-chunk-11-3.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="DataAnalysis_files/figure-html/unnamed-chunk-11-4.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="DataAnalysis_files/figure-html/unnamed-chunk-11-5.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="DataAnalysis_files/figure-html/unnamed-chunk-11-6.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="DataAnalysis_files/figure-html/unnamed-chunk-11-7.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="DataAnalysis_files/figure-html/unnamed-chunk-11-8.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="DataAnalysis_files/figure-html/unnamed-chunk-11-9.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="DataAnalysis_files/figure-html/unnamed-chunk-11-10.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="DataAnalysis_files/figure-html/unnamed-chunk-11-11.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="DataAnalysis_files/figure-html/unnamed-chunk-11-12.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="DataAnalysis_files/figure-html/unnamed-chunk-11-13.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="DataAnalysis_files/figure-html/unnamed-chunk-11-14.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="DataAnalysis_files/figure-html/unnamed-chunk-11-15.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="DataAnalysis_files/figure-html/unnamed-chunk-11-16.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="DataAnalysis_files/figure-html/unnamed-chunk-11-17.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="DataAnalysis_files/figure-html/unnamed-chunk-11-18.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="DataAnalysis_files/figure-html/unnamed-chunk-11-19.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="DataAnalysis_files/figure-html/unnamed-chunk-11-20.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="DataAnalysis_files/figure-html/unnamed-chunk-11-21.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="DataAnalysis_files/figure-html/unnamed-chunk-11-22.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</section>
<section id="test-plot" class="level3">
<h3 class="anchored" data-anchor-id="test-plot">test plot</h3>
<div class="cell">
<details class="code-fold">
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb14"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb14-1"><a href="#cb14-1" aria-hidden="true" tabindex="-1"></a>t <span class="sc">%&gt;%</span> </span>
<span id="cb14-2"><a href="#cb14-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">SummedIntensity =</span> <span class="fu">log10</span>(SummedIntensity)) <span class="sc">%&gt;%</span> </span>
<span id="cb14-3"><a href="#cb14-3" aria-hidden="true" tabindex="-1"></a><span class="fu">ggplot</span>(<span class="fu">aes</span>(SummedIntensity, R.FileName, <span class="at">fill=</span>R.Condition, <span class="at">color=</span>R.Condition))<span class="sc">+</span></span>
<span id="cb14-4"><a href="#cb14-4" aria-hidden="true" tabindex="-1"></a><span class="fu">geom_col</span>()<span class="sc">+</span></span>
<span id="cb14-5"><a href="#cb14-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">title=</span><span class="st">""</span>, <span class="at">subtitle=</span><span class="st">""</span>, <span class="at">x=</span><span class="st">""</span>, <span class="at">y=</span><span class="st">""</span>)<span class="sc">+</span></span>
<span id="cb14-6"><a href="#cb14-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_minimal</span>()<span class="sc">+</span></span>
<span id="cb14-7"><a href="#cb14-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(<span class="at">legend.position =</span> <span class="st">"none"</span>,</span>
<span id="cb14-8"><a href="#cb14-8" aria-hidden="true" tabindex="-1"></a>        <span class="at">axis.text.y =</span> <span class="fu">element_blank</span>())</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="DataAnalysis_files/figure-html/unnamed-chunk-12-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</section>
<section id="protein-intensity-distributions" class="level3">
<h3 class="anchored" data-anchor-id="protein-intensity-distributions">Protein intensity distributions</h3>
<div class="cell">
<details class="code-fold">
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb15"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb15-1"><a href="#cb15-1" aria-hidden="true" tabindex="-1"></a>dfBGS <span class="sc">%&gt;%</span> </span>
<span id="cb15-2"><a href="#cb15-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">MDBL_PlotIntensityDensity</span>()</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="DataAnalysis_files/figure-html/unnamed-chunk-13-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</section>
<section id="cv-for-each-protein-in-condition" class="level3">
<h3 class="anchored" data-anchor-id="cv-for-each-protein-in-condition">CV for each protein in condition</h3>
<div class="cell">
<details class="code-fold">
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb16"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb16-1"><a href="#cb16-1" aria-hidden="true" tabindex="-1"></a>dfBGS <span class="sc">%&gt;%</span> </span>
<span id="cb16-2"><a href="#cb16-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">MDBL_PlotCV</span>()</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="DataAnalysis_files/figure-html/unnamed-chunk-14-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</section>
<section id="cv-for-each-protein-in-condition-with-vv-filtering" class="level3">
<h3 class="anchored" data-anchor-id="cv-for-each-protein-in-condition-with-vv-filtering">CV for each protein in condition with VV filtering</h3>
<div class="cell">
<details class="code-fold">
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb17"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb17-1"><a href="#cb17-1" aria-hidden="true" tabindex="-1"></a>dfBGS <span class="sc">%&gt;%</span></span>
<span id="cb17-2"><a href="#cb17-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">MDBL_FilterVV</span>(<span class="fl">0.2</span>) <span class="sc">%&gt;%</span> </span>
<span id="cb17-3"><a href="#cb17-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">MDBL_PlotCV</span>()</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="DataAnalysis_files/figure-html/unnamed-chunk-15-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</section>
<section id="data-completeness" class="level3">
<h3 class="anchored" data-anchor-id="data-completeness">Data completeness</h3>
<div class="cell">
<details class="code-fold">
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb18"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb18-1"><a href="#cb18-1" aria-hidden="true" tabindex="-1"></a>dfBGS <span class="sc">%&gt;%</span> </span>
<span id="cb18-2"><a href="#cb18-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">MDBL_DataCompleteness</span>()</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="DataAnalysis_files/figure-html/unnamed-chunk-16-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</section>
<section id="s-curve" class="level3">
<h3 class="anchored" data-anchor-id="s-curve">S-curve</h3>
<div class="cell">
<details class="code-fold">
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb19"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb19-1"><a href="#cb19-1" aria-hidden="true" tabindex="-1"></a>dfBGS <span class="sc">%&gt;%</span> </span>
<span id="cb19-2"><a href="#cb19-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">distinct</span>(R.Condition, PG.ProteinAccessions, R.FileName, PG.Quantity) <span class="sc">%&gt;%</span> </span>
<span id="cb19-3"><a href="#cb19-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(<span class="sc">!</span><span class="fu">is.na</span>(PG.Quantity)) <span class="sc">%&gt;%</span> </span>
<span id="cb19-4"><a href="#cb19-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">PG.Quantity =</span> <span class="fu">log</span>(<span class="at">base=</span><span class="dv">10</span>, PG.Quantity)) <span class="sc">%&gt;%</span> </span>
<span id="cb19-5"><a href="#cb19-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(R.FileName) <span class="sc">%&gt;%</span> </span>
<span id="cb19-6"><a href="#cb19-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">arrange</span>(<span class="fu">desc</span>(PG.Quantity)) <span class="sc">%&gt;%</span> </span>
<span id="cb19-7"><a href="#cb19-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">n=</span><span class="fu">row_number</span>()) <span class="sc">%&gt;%</span> </span>
<span id="cb19-8"><a href="#cb19-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ungroup</span>() <span class="sc">%&gt;%</span> </span>
<span id="cb19-9"><a href="#cb19-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ggplot</span>(<span class="fu">aes</span>(n, PG.Quantity, <span class="at">group=</span>R.FileName, <span class="at">color=</span>R.Condition))<span class="sc">+</span></span>
<span id="cb19-10"><a href="#cb19-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_line</span>()<span class="sc">+</span></span>
<span id="cb19-11"><a href="#cb19-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">facet_wrap</span>(<span class="sc">~</span>R.Condition, <span class="at">scales =</span> <span class="st">"free_x"</span>)<span class="sc">+</span></span>
<span id="cb19-12"><a href="#cb19-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">title=</span><span class="st">"Intensity distribution, S-curve"</span>, <span class="at">x=</span><span class="st">"proteins"</span>, <span class="at">y=</span><span class="st">"[log10]Intensity"</span>)<span class="sc">+</span></span>
<span id="cb19-13"><a href="#cb19-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_minimal</span>()<span class="sc">+</span></span>
<span id="cb19-14"><a href="#cb19-14" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(<span class="at">legend.position =</span> <span class="st">"none"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="DataAnalysis_files/figure-html/unnamed-chunk-17-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</section>
<section id="peptides-per-protein" class="level3">
<h3 class="anchored" data-anchor-id="peptides-per-protein">Peptides per protein</h3>
<p>This is based on a grouping per protein in each group. So a 1 means a protein identified with only one peptide for all samples in this group.</p>
<div class="cell">
<details class="code-fold">
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb20"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb20-1"><a href="#cb20-1" aria-hidden="true" tabindex="-1"></a>dfBGS <span class="sc">%&gt;%</span> </span>
<span id="cb20-2"><a href="#cb20-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">distinct</span>(R.Condition, PG.ProteinAccessions, EG.ModifiedPeptide) <span class="sc">%&gt;%</span></span>
<span id="cb20-3"><a href="#cb20-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(R.Condition, PG.ProteinAccessions) <span class="sc">%&gt;%</span> </span>
<span id="cb20-4"><a href="#cb20-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">tally</span>(<span class="at">name =</span> <span class="st">"peptides_per_protein"</span>) <span class="sc">%&gt;%</span> </span>
<span id="cb20-5"><a href="#cb20-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">arrange</span>(peptides_per_protein) <span class="sc">%&gt;%</span> </span>
<span id="cb20-6"><a href="#cb20-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(R.Condition) <span class="sc">%&gt;%</span> </span>
<span id="cb20-7"><a href="#cb20-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">row =</span> <span class="fu">row_number</span>()) <span class="sc">%&gt;%</span> </span>
<span id="cb20-8"><a href="#cb20-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ggplot</span>(<span class="fu">aes</span>(row, peptides_per_protein, <span class="at">color=</span>R.Condition))<span class="sc">+</span></span>
<span id="cb20-9"><a href="#cb20-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_step</span>()<span class="sc">+</span></span>
<span id="cb20-10"><a href="#cb20-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_y_log10</span>()<span class="sc">+</span></span>
<span id="cb20-11"><a href="#cb20-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">facet_wrap</span>(<span class="sc">~</span>R.Condition, <span class="at">scale =</span> <span class="st">"free"</span>)<span class="sc">+</span></span>
<span id="cb20-12"><a href="#cb20-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">title=</span><span class="st">"Unique modified peptides per protein in group"</span>, <span class="at">x=</span><span class="st">"proteins"</span>, <span class="at">y=</span><span class="st">"peptides"</span>)<span class="sc">+</span></span>
<span id="cb20-13"><a href="#cb20-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_minimal</span>()<span class="sc">+</span></span>
<span id="cb20-14"><a href="#cb20-14" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(<span class="at">legend.position =</span> <span class="st">"none"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="DataAnalysis_files/figure-html/unnamed-chunk-18-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</section>
<section id="peptides-per-protein-per-raw-file" class="level3">
<h3 class="anchored" data-anchor-id="peptides-per-protein-per-raw-file">Peptides per protein per raw file</h3>
<p>This is based on average number of peptides per protein across groups. So a 1 means that a given protein is only identified with 1 peptide on average across all samples in group.</p>
<div class="cell">
<details class="code-fold">
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb21"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb21-1"><a href="#cb21-1" aria-hidden="true" tabindex="-1"></a>dfBGS <span class="sc">%&gt;%</span> </span>
<span id="cb21-2"><a href="#cb21-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">distinct</span>(R.FileName, R.Condition, PG.ProteinAccessions, EG.ModifiedPeptide) <span class="sc">%&gt;%</span></span>
<span id="cb21-3"><a href="#cb21-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(R.FileName, R.Condition, PG.ProteinAccessions) <span class="sc">%&gt;%</span> </span>
<span id="cb21-4"><a href="#cb21-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">tally</span>(<span class="at">name =</span> <span class="st">"peptides_per_protein"</span>) <span class="sc">%&gt;%</span> </span>
<span id="cb21-5"><a href="#cb21-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ungroup</span>() <span class="sc">%&gt;%</span> </span>
<span id="cb21-6"><a href="#cb21-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(R.Condition, PG.ProteinAccessions) <span class="sc">%&gt;%</span> </span>
<span id="cb21-7"><a href="#cb21-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">summarise</span>(<span class="at">n =</span> <span class="fu">n</span>(),</span>
<span id="cb21-8"><a href="#cb21-8" aria-hidden="true" tabindex="-1"></a>            <span class="at">mean =</span> <span class="fu">mean</span>(peptides_per_protein)) <span class="sc">%&gt;%</span> </span>
<span id="cb21-9"><a href="#cb21-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">arrange</span>(mean) <span class="sc">%&gt;%</span> </span>
<span id="cb21-10"><a href="#cb21-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(R.Condition) <span class="sc">%&gt;%</span> </span>
<span id="cb21-11"><a href="#cb21-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">row =</span> <span class="fu">row_number</span>()) <span class="sc">%&gt;%</span> </span>
<span id="cb21-12"><a href="#cb21-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ggplot</span>(<span class="fu">aes</span>(row, mean, <span class="at">color=</span>R.Condition))<span class="sc">+</span></span>
<span id="cb21-13"><a href="#cb21-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_step</span>()<span class="sc">+</span></span>
<span id="cb21-14"><a href="#cb21-14" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_y_log10</span>()<span class="sc">+</span></span>
<span id="cb21-15"><a href="#cb21-15" aria-hidden="true" tabindex="-1"></a>  <span class="fu">facet_wrap</span>(<span class="sc">~</span>R.Condition, <span class="at">scale =</span> <span class="st">"free"</span>)<span class="sc">+</span></span>
<span id="cb21-16"><a href="#cb21-16" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">title=</span><span class="st">"Average number of unique modified peptides per protein per sample in group"</span>, <span class="at">x=</span><span class="st">"proteins"</span>, <span class="at">y=</span><span class="st">"peptides"</span>)<span class="sc">+</span></span>
<span id="cb21-17"><a href="#cb21-17" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_minimal</span>()<span class="sc">+</span></span>
<span id="cb21-18"><a href="#cb21-18" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(<span class="at">legend.position =</span> <span class="st">"none"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="DataAnalysis_files/figure-html/unnamed-chunk-19-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</section>
<section id="plasma-qc" class="level3">
<h3 class="anchored" data-anchor-id="plasma-qc">Plasma QC</h3>
<div class="cell">
<details class="code-fold">
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb22"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb22-1"><a href="#cb22-1" aria-hidden="true" tabindex="-1"></a>dfPlasmaQC <span class="ot">&lt;-</span> <span class="fu">read.csv</span>(<span class="st">"PlasmaQC/CleanList.csv"</span>, <span class="at">sep=</span><span class="st">";"</span>) <span class="sc">%&gt;%</span> </span>
<span id="cb22-2"><a href="#cb22-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">rename</span>(<span class="at">PG.Genes =</span> Gene.names)</span>
<span id="cb22-3"><a href="#cb22-3" aria-hidden="true" tabindex="-1"></a>cutoff_sd <span class="ot">=</span> <span class="dv">2</span></span>
<span id="cb22-4"><a href="#cb22-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb22-5"><a href="#cb22-5" aria-hidden="true" tabindex="-1"></a><span class="co">#plot</span></span>
<span id="cb22-6"><a href="#cb22-6" aria-hidden="true" tabindex="-1"></a>dfBGS <span class="sc">%&gt;%</span></span>
<span id="cb22-7"><a href="#cb22-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(<span class="sc">!</span><span class="fu">str_detect</span>(R.FileName, <span class="st">"ET_QC"</span>)) <span class="sc">%&gt;%</span> </span>
<span id="cb22-8"><a href="#cb22-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">select</span>(R.FileName, PG.Genes, PG.Quantity, R.Condition) <span class="sc">%&gt;%</span> </span>
<span id="cb22-9"><a href="#cb22-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">PG.Genes =</span> <span class="fu">toupper</span>(PG.Genes)) <span class="sc">%&gt;%</span> </span>
<span id="cb22-10"><a href="#cb22-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">PG.Genes =</span> <span class="fu">str_remove</span>(PG.Genes, <span class="st">";.*"</span>)) <span class="sc">%&gt;%</span> </span>
<span id="cb22-11"><a href="#cb22-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">left_join</span>(dfPlasmaQC, <span class="at">by =</span> <span class="fu">join_by</span>(PG.Genes), <span class="at">relationship =</span> <span class="st">"many-to-many"</span>) <span class="sc">%&gt;%</span> </span>
<span id="cb22-12"><a href="#cb22-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">Type =</span> <span class="fu">as.character</span>(Type)) <span class="sc">%&gt;%</span> </span>
<span id="cb22-13"><a href="#cb22-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">Type =</span> <span class="fu">replace_na</span>(Type, <span class="st">"Plasma"</span>)) <span class="sc">%&gt;%</span> </span>
<span id="cb22-14"><a href="#cb22-14" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(R.FileName, Type) <span class="sc">%&gt;%</span> </span>
<span id="cb22-15"><a href="#cb22-15" aria-hidden="true" tabindex="-1"></a>  <span class="fu">summarise</span>(<span class="at">sum =</span> <span class="fu">sum</span>(PG.Quantity, <span class="at">na.rm =</span> T)) <span class="sc">%&gt;%</span> </span>
<span id="cb22-16"><a href="#cb22-16" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">sum =</span> <span class="fu">log10</span>(sum)) <span class="sc">%&gt;%</span> </span>
<span id="cb22-17"><a href="#cb22-17" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ungroup</span>() <span class="sc">%&gt;%</span></span>
<span id="cb22-18"><a href="#cb22-18" aria-hidden="true" tabindex="-1"></a>  <span class="fu">pivot_wider</span>(<span class="at">names_from =</span> Type, <span class="at">values_from =</span> sum) <span class="sc">%&gt;%</span> </span>
<span id="cb22-19"><a href="#cb22-19" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">erythrocyte_Index =</span> erythrocyte<span class="sc">/</span>Plasma,</span>
<span id="cb22-20"><a href="#cb22-20" aria-hidden="true" tabindex="-1"></a>         <span class="at">platelet_Index =</span> platelet<span class="sc">/</span>Plasma,</span>
<span id="cb22-21"><a href="#cb22-21" aria-hidden="true" tabindex="-1"></a>         <span class="co">#coagulationSerum_Index = Plasma/coagulation_UpSerum,</span></span>
<span id="cb22-22"><a href="#cb22-22" aria-hidden="true" tabindex="-1"></a>         <span class="at">coagulationPlasma_Index =</span> Plasma<span class="sc">/</span>coagulation_UpPlasma) <span class="sc">%&gt;%</span> </span>
<span id="cb22-23"><a href="#cb22-23" aria-hidden="true" tabindex="-1"></a>  <span class="fu">select</span>(R.FileName, <span class="fu">contains</span>(<span class="st">"_Index"</span>)) <span class="sc">%&gt;%</span> </span>
<span id="cb22-24"><a href="#cb22-24" aria-hidden="true" tabindex="-1"></a>  <span class="co">#select(name, contains("platelet_Index")) %&gt;% </span></span>
<span id="cb22-25"><a href="#cb22-25" aria-hidden="true" tabindex="-1"></a>  <span class="fu">pivot_longer</span>(<span class="at">names_to =</span>  <span class="st">"key"</span>, <span class="at">values_to =</span>  <span class="st">"val"</span>, <span class="sc">-</span>R.FileName) <span class="sc">%&gt;%</span> </span>
<span id="cb22-26"><a href="#cb22-26" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(<span class="sc">!</span><span class="fu">is.na</span>(val)) <span class="sc">%&gt;%</span> </span>
<span id="cb22-27"><a href="#cb22-27" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(key) <span class="sc">%&gt;%</span> </span>
<span id="cb22-28"><a href="#cb22-28" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">sd =</span> <span class="fu">sd</span>(val, <span class="at">na.rm =</span> T),</span>
<span id="cb22-29"><a href="#cb22-29" aria-hidden="true" tabindex="-1"></a>         <span class="at">mean =</span> <span class="fu">mean</span>(val, <span class="at">na.rm =</span> T),</span>
<span id="cb22-30"><a href="#cb22-30" aria-hidden="true" tabindex="-1"></a>         <span class="at">outlier =</span> <span class="fu">ifelse</span>(val<span class="sc">&gt;</span>(mean<span class="sc">+</span>cutoff_sd<span class="sc">*</span>sd),<span class="cn">TRUE</span>,<span class="cn">FALSE</span>),</span>
<span id="cb22-31"><a href="#cb22-31" aria-hidden="true" tabindex="-1"></a>         <span class="at">label_outlier =</span> <span class="fu">ifelse</span>(outlier<span class="sc">==</span><span class="cn">TRUE</span>,R.FileName,<span class="st">""</span>)) <span class="ot">-&gt;</span> dfTemp</span>
<span id="cb22-32"><a href="#cb22-32" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb22-33"><a href="#cb22-33" aria-hidden="true" tabindex="-1"></a>dfTemp <span class="sc">%&gt;%</span> </span>
<span id="cb22-34"><a href="#cb22-34" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ggplot</span>(<span class="fu">aes</span>(R.FileName,val, <span class="at">fill=</span>outlier))<span class="sc">+</span></span>
<span id="cb22-35"><a href="#cb22-35" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_col</span>(<span class="at">show.legend =</span> F, <span class="at">width =</span> <span class="dv">1</span>)<span class="sc">+</span></span>
<span id="cb22-36"><a href="#cb22-36" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_text</span>(<span class="fu">aes</span>(<span class="at">label =</span> label_outlier), <span class="at">hjust =</span> <span class="st">"inward"</span>,  <span class="at">size=</span><span class="dv">3</span>)<span class="sc">+</span></span>
<span id="cb22-37"><a href="#cb22-37" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_fill_manual</span>(<span class="at">values =</span> <span class="fu">c</span>(<span class="st">"grey80"</span>, <span class="st">"red"</span>)) <span class="sc">+</span> </span>
<span id="cb22-38"><a href="#cb22-38" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_hline</span>(<span class="fu">aes</span>(<span class="at">yintercept =</span> mean <span class="sc">+</span> (<span class="dv">1</span><span class="sc">*</span>sd)), <span class="at">color=</span><span class="st">"blue"</span>, <span class="at">linetype =</span> <span class="st">"dashed"</span>)<span class="sc">+</span> <span class="co">#plus 1 sd</span></span>
<span id="cb22-39"><a href="#cb22-39" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_hline</span>(<span class="fu">aes</span>(<span class="at">yintercept =</span> mean <span class="sc">+</span> (<span class="dv">2</span><span class="sc">*</span>sd)), <span class="at">color=</span><span class="st">"blue"</span>, <span class="at">linetype =</span> <span class="st">"dashed"</span>)<span class="sc">+</span> <span class="co">#plus 2 sd</span></span>
<span id="cb22-40"><a href="#cb22-40" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_hline</span>(<span class="fu">aes</span>(<span class="at">yintercept =</span> mean <span class="sc">+</span> (<span class="dv">3</span><span class="sc">*</span>sd)), <span class="at">color=</span><span class="st">"blue"</span>, <span class="at">linetype =</span> <span class="st">"dashed"</span>)<span class="sc">+</span> <span class="co">#plus 3 sd</span></span>
<span id="cb22-41"><a href="#cb22-41" aria-hidden="true" tabindex="-1"></a>  <span class="fu">facet_wrap</span>(<span class="sc">~</span>key, <span class="at">scales =</span> <span class="st">"free_x"</span>, <span class="at">ncol=</span><span class="dv">3</span>)<span class="sc">+</span></span>
<span id="cb22-42"><a href="#cb22-42" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_minimal</span>()<span class="sc">+</span></span>
<span id="cb22-43"><a href="#cb22-43" aria-hidden="true" tabindex="-1"></a>  <span class="fu">coord_flip</span>()<span class="sc">+</span></span>
<span id="cb22-44"><a href="#cb22-44" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">title=</span><span class="st">"Plasma Proteome Profiling QC"</span>, <span class="at">subtitle=</span><span class="fu">paste</span>(cutoff_sd,<span class="st">"x sd cutoff"</span>, <span class="at">sep =</span> <span class="st">""</span>), <span class="at">x=</span><span class="st">""</span>, <span class="at">y=</span><span class="st">""</span>)<span class="sc">+</span></span>
<span id="cb22-45"><a href="#cb22-45" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_minimal</span>()<span class="sc">+</span></span>
<span id="cb22-46"><a href="#cb22-46" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(<span class="at">panel.grid.major =</span> <span class="fu">element_blank</span>(), </span>
<span id="cb22-47"><a href="#cb22-47" aria-hidden="true" tabindex="-1"></a>        <span class="at">panel.grid.minor =</span> <span class="fu">element_blank</span>(),</span>
<span id="cb22-48"><a href="#cb22-48" aria-hidden="true" tabindex="-1"></a>        <span class="at">axis.text.y =</span> <span class="fu">element_blank</span>())</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="DataAnalysis_files/figure-html/unnamed-chunk-20-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
<details class="code-fold">
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb23"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb23-1"><a href="#cb23-1" aria-hidden="true" tabindex="-1"></a>dfTemp <span class="sc">%&gt;%</span> </span>
<span id="cb23-2"><a href="#cb23-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(outlier <span class="sc">==</span> T) <span class="sc">%&gt;%</span> </span>
<span id="cb23-3"><a href="#cb23-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">arrange</span>(key) </span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output cell-output-stdout">
<pre><code># A tibble: 7 × 7
# Groups:   key [3]
  R.FileName                      key     val     sd  mean outlier label_outlier
  &lt;chr&gt;                           &lt;chr&gt; &lt;dbl&gt;  &lt;dbl&gt; &lt;dbl&gt; &lt;lgl&gt;   &lt;chr&gt;        
1 20250624_GJAL_AS_Evo_8cm_100SP… coag… 1.24  0.0602 1.12  TRUE    20250624_GJA…
2 20250624_GJAL_AS_Evo_8cm_100SP… coag… 1.26  0.0602 1.12  TRUE    20250624_GJA…
3 20250624_GJAL_AS_Evo_8cm_100SP… eryt… 0.661 0.0282 0.602 TRUE    20250624_GJA…
4 20250624_GJAL_AS_Evo_8cm_100SP… eryt… 0.658 0.0282 0.602 TRUE    20250624_GJA…
5 20250624_GJAL_AS_Evo_8cm_100SP… eryt… 0.686 0.0282 0.602 TRUE    20250624_GJA…
6 20250624_GJAL_AS_Evo_8cm_100SP… plat… 0.760 0.0225 0.698 TRUE    20250624_GJA…
7 20250624_GJAL_AS_Evo_8cm_100SP… plat… 0.755 0.0225 0.698 TRUE    20250624_GJA…</code></pre>
</div>
</div>
</section>
<section id="heatmap" class="level3">
<h3 class="anchored" data-anchor-id="heatmap">Heatmap</h3>
<div class="cell">
<details class="code-fold">
<summary>Show the code</summary>
<div class="sourceCode cell-code" id="cb25"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb25-1"><a href="#cb25-1" aria-hidden="true" tabindex="-1"></a>dfBGS <span class="sc">%&gt;%</span> </span>
<span id="cb25-2"><a href="#cb25-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">MDBL_HeatMap</span>()</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="DataAnalysis_files/figure-html/unnamed-chunk-21-1.png" class="img-fluid figure-img" width="672"></p>
</figure>
</div>
</div>
</div>
</section>
<section id="umap-not-working" class="level3">
<h3 class="anchored" data-anchor-id="umap-not-working">UMAP, not working</h3>
</section>
</section>

</main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->




</body></html>